Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM rust:1.79.0 AS yara_builder_base
+FROM amazonlinux:2 AS builder_base
```

Comment:
Amazon Linux is a RHEL-based distribution. Couldn't we use Alma 8 as our base image, so that we have a more 'standard' base image?

Reply:
Also Amazon Linux 2 is now deprecated, since they rolled out Amazon Linux 2023. Another reason to use Alma 😉

Reply:
Maybe? IDK I'd need to test this. In theory it should work because Alma 8 and RHEL 8 have the same glibc version.